compiler/backend/presLangScript.sml

```diff
@@ -33,8 +39,7 @@ val _ = Datatype`
     | Plit lit
     (* TODO: Consider doing what we did with op above for the patterns below, in
```

Comment:
Not sure as to whether this comment should remain here. I guess it is something to consider for future language implementations. I believe the comment originates from separating the Con-datatype found in Ast and in conLang.